src/20_Components/amp-font.html

```diff
@@ -0,0 +1,65 @@
+<!-- 
+  #### Introduction
+
+  `amp-font` allows to control timeouts on font loading on AMP HTML pages.
+
+  AMP HTML files don't support the normal HTML `img` tag. With `amp-img` AMP
+  provides a powerful replacement.
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="https://ampbyexample.com/components/amp-font/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-font" src="https://cdn.ampproject.org/v0/amp-font-0.1.js"></script>
+  <!-- 
+    Define some CSS classes so that `amp-font` can add or remove those based on whether a font was loaded or is in error-state. Find out about all the supported css classes [here](https://github.com/ampproject/amphtml/blob/b1607dff9446d332afcdead64b63e8d25e74cff7/extensions/amp-font/amp-font.md)
+  -->
+  <style amp-custom>
+    @font-face {
+      font-family: 'Comic AMP';
+      font-style: normal;
+      font-weight: 400;
+      src: url(fonts/ComicAMP.ttf) format('truetype');
+    }
+    .comic-amp-font-loaded .comic-amp{
+      font-family: 'Comic AMP', serif, sans-serif;
+    }
+    .comic-amp-font-loading {
+      color: #00f;
+    }
+    .comic-amp-font-loading .comic-amp {
+      color: #0ff;
+    }  
+    .comic-amp-font-missing .comic-amp {
+      color: red;
+    }
+  </style>
+</head>
+<body class="comic-amp-font-loading comic-amp-bold-font-loading">
+  <!-- #### Basic Usage -->
+  <!--
+    A simple text showing how to use `amp-font`.
+  -->
+  <p class="comic-amp">
+    The following text will be displayed in red if the chosen font is not supported.
+  </p>
+  <p class="comic-amp">
+    "Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit..."
+    "There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain..."
+  </p>
+
+  <amp-font
```

Comment:
Added some more explanation. I would avoid to copy the description of all the attributes and refer to the official docs, it's more maintanable.